### PR TITLE
Stuff from the RPC review of RFC 9000

### PR DESCRIPTION
**Pull request by martinthomson**
Individual commits break this down, but this is all fairly prosaic stuff..

### CODE REVIEW

**Review by ianswett — APPROVED** [review]

**Review by nibanks — APPROVED** [review]

File: rfc9002.md [hunk 1]
```diff
@@ -367,7 +367,7 @@ smoothed_rtt.
 Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent
 congestion is established. This avoids repeatedly declaring persistent
 congestion when the RTT increases. This also allows a connection to reset
-its estimate of min_rtt and smoothed_rtt after a disruptive network event;
+its estimate of min_rtt and smoothed_rtt after a disruptive network event;
```
Comment (janaiyengar) [inline rfc9002.md:370]: I like pink spaces better.

File: rfc9000.md [hunk 2]
```diff
@@ -6916,10 +6915,10 @@ that are received on a different network path.
 The Source and Destination Connection ID fields are the primary means of
 protection against an off-path attack during the handshake; see
 {{validate-handshake}}.  These are required to match those set by a peer.
-Except for Initial and Stateless Reset packets, an endpoint only accepts
-packets that include a Destination Connection ID field that matches a value the
-endpoint previously chose.  This is the only protection offered for Version
-Negotiation packets.
+Except for Initial and Stateless Resets, an endpoint only accepts packets that
```
Comment (janaiyengar) [inline rfc9000.md:6918]: I'm not convinced that this is more consistent (here and all the Stateless Reset changes above). Just at the bottom of this paragraph, we say Version Negotiation packets. Given that the capital case represents packet types, what is the argument for removing packets? I'll admit it's redundant, but we use it quite a bit elsewhere, and I believe that it'd be more consistent to use `Stateless Reset packets`.

**Review by janaiyengar — COMMENTED** [review]
I'm not sure of the Stateless Reset changes. See inline.

File: rfc9002.md (hunk 1 above)
Comment (martinthomson) [reply rfc9002.md:370]: My hex decoder tells me that this is non-breaking space.  How that got there is ... interesting.

File: rfc9000.md (hunk 2 above)
Comment (martinthomson) [reply rfc9000.md:6918]: Ah, but Version Negotiation is a packet (it is defined thus).  Stateless Reset is defined as a UDP datagram, not a packet.  That's why (at least in my opinion) the change is necessary.

**Review by janaiyengar — APPROVED** [review]
Thanks for clarifying and for fixing the one awkward case.

File: rfc9000.md [hunk 3]
```diff
@@ -4345,7 +4344,7 @@ injection as specified in {{!RFC8201}} and {{Section 5.2 of RFC8085}}.  This
 validation SHOULD use the quoted packet supplied in the payload of an ICMP
 message to associate the message with a corresponding transport connection (see
 {{Section 4.6.1 of DPLPMTUD}}).  ICMP message validation MUST include matching
-IP addresses and UDP ports {{!BCP145}} and, when possible, connection IDs to
+IP addresses and UDP ports {{!RFC8085}} and, when possible, connection IDs to
```
Comment (MikeBishop) [inline rfc9000.md:4347]: Not related to this change, but why is this a normative reference and line 4305 informative?

**Pull request merged**